newsApi.js

```diff
@@ -1,8 +1,9 @@
 class NewsApi {
-  loadNews() {
+  loadNews(callback) {
     fetch('http://content.guardianapis.com/politics/blog/2014/feb/17/alex-salmond-speech-first-minister-scottish-independence-eu-currency-live?show-fields=body&api-key=98e51f22-b91b-4a89-b784-aa14c541dfed')
```

Comment:
To abstract the API key, it's worth using a package called webpack, which is basically an alternative to esbuild. Esbuild kept messing things up so I decided to switch gears. Webpack can be configured by creating a webpack.config.js file, which basically instructs bundle.js to refer to .env files throughout your project.